databases/week2/exercise1-2.sql

```diff
@@ -0,0 +1,77 @@
+/*Working with tasks*/
+use hyf_lesson2;
+/*Add a task with these attributes: 
+title, description, created, updated, due_date, status_id, user_id*/
+INSERT INTO
+  task (
+    id,
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id
+  )
+VALUES
+  (
+    36,
+    'Meditation',
+    'As less 5min',
+    '2021-10-25 06:54:16',
+    '2021-10-15 13:05:09',
+    '2021-12-15 13:05:09',
+    2
+  );
+  /*Change the title of a task*/
+UPDATE
+  task
+SET
+  title = "Do yoga exercise"
+WHERE
+  id = 36;
+  /*Change a task due date*/
+UPDATE
+  task
+SET
+  due_date = '2021-11-05'
+WHERE
+  id = 36;
+  /*Change a task status*/
+UPDATE
+  task
+SET
+  status_id = "3"
+WHERE
+  id = 6;
+  /*Mark a task as complete*/
+UPDATE
+  task
+SET
+  status_id = 3
+where
+  id = 36;
+--Delete a task
+DELETE FROM
+  task
+WHERE
+  id = 36;
+
+-----/*School database*/------
+
+/*Create a new database containing the following tables:
+Class: with the columns: id, name, begins (date), ends (date)
+Student: with the columns: id, name, email, phone, class_id (foreign key)*/
+
+CREATE Table class (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, begins date, ends date); 
```

Comment:
Note for the future, to be easier to read, please format it like [here](https://github.com/HackYourFuture-CPH/databases/blob/class18/lesson1/lesson-data.sql) :)

Also, what's the character set?
I'm guessing that if you don't specify it and you try to insert chinese or arabic letters it will only show `???`...